// Copyright 2023 The Chromium Authors
// Use of this source code is governed by a BSD-style license that can be
// found in the LICENSE file.

syntax = "proto3";

option optimize_for = LITE_RUNTIME;
option java_package = "org.chromium.components.optimization_guide.proto";
option java_outer_classname = "ClientSidePhishingModelMetadataProto";

package optimization_guide.proto;

// The message contains an integer to pair with a client side phishing image
// embedding model which will also have an integer to match.
message ClientSidePhishingModelMetadata {
  optional int32 image_embedding_model_version = 1;
}
